Packs/Lumu/Integrations/Lumu/Lumu.py

```diff
@@ -0,0 +1,1080 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+""" IMPORTS """
+
+
+import hashlib
+import urllib3
+import hmac
+from collections import Counter
+
+""" CONSTANTS """
+
+""" CLIENT CLASS """
+
+""" HELPER FUNCTIONS """
+
+""" COMMAND FUNCTIONS """
+
+""" MAIN FUNCTION """
+
+
+params = demisto.params()
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.000Z"
+MIRROR_DIRECTION = {
+    "Incoming": "In",
+    "Outgoing": "Out",
+    "Incoming And Outgoing": "Both",
+}.get(params.get("mirror_direction"))
+MIRROR_TAGS = params.get("mirror_tags") or []
+INTEGRATION_INSTANCE = demisto.integrationInstance()
+FIELDS_TO_REMOVE_FROM_MIROR_IN = ["url", "id", "created_at"]
+
+status_msg_types = [
+    "IncidentMuted",
+    "IncidentUnmuted",
+    "IncidentClosed",
+    "IncidentCommentAdded",
+]
+
+
+class Client(BaseClient):
+    def __init__(self, server_url, verify, proxy, headers, auth):
```

Comment:
```suggestion
    def __init__(self, server_url, verify, proxy):
```

Reply:
sort it out in commit 2bb633df7de2b53b93d8a8cdc1fcaef13e33c6a7